.erb-lint.yml

```diff
@@ -60,7 +60,6 @@ linters:
           - '((sm|md|lg)-)?hide'
           - '(sm|md|lg)-show'
           - 'btn(-(small|big|narrow|wide|link|primary|secondary|danger|disabled|big|narrow|transparent))?'
-          - ':not(label).btn-border'
```

Comment:
is this removed because we don't use the `btn-border` style any more? or because the style no longer exists?

Reply:
> is this removed because we don't use the `btn-border` style any more? or because the style no longer exists?

I could have kept it and removed the one exception we'd previously allowed:

```yml
- '.btn-border'
```

This would also be most consistent with how we've used the deprecated class linting to date. I think I've just become less convinced over time of the value of these deprecations. Because the class no longer exists, it'll be obviously visually broken if someone tries to use it.  It really only prevents a case where someone is copy-pasting code, which itself is unlikely because there will no longer be any remnants of the class in the codebase after its removal.

That being said, I'll probably re-add it, just for consistency's sake, and we can decide separately if / how to deal with deprecations.

Reply:
> That being said, I'll probably re-add it, just for consistency's sake, and we can decide separately if / how to deal with deprecations.

Re-added in e519dc7ec.